api/resources/translations/messages-fr.properties

```
@@ -720,61 +763,94 @@ instance.upgrade = Mise à niveau instance
 instance.upgrade.at = à
 instance.upgrade.betas = Bétas
 instance.upgrade.branches = Branches
+instance.upgrade.build.version = Version
+instance.upgrade.cancel = Annuler la mise à jour
+instance.upgrade.cancelling = Annulation de la mise à jour en cours
+instance.upgrade.cancel.note = L'annulation arrêtera la mise à jour en cours et réinitialisera toute progression effectuée jusqu'à présent.
+instance.upgrade.cancel.summary = Vous êtes sur le point d'annuler la mise à jour de {{before}} vers {{after}}.
 instance.upgrade.complete = Déploiement complet
-instance.upgrade.confirm = Confirmer le upgrade
+instance.upgrade.confirm = Confirmer la mise à jour
+instance.upgrade.confirming = Confirmation de la mise à jour
 instance.upgrade.confirm.summary = Nous allons faire le upgrade de {{before}} à {{after}}.
 instance.upgrade.confirm.warning.text = Cette action ne peut pas être annulée, assurez-vous que vos données ont été sauvegardées\!
 instance.upgrade.current_version = Version actuelle
-instance.upgrade.date =
+instance.upgrade.date = Date de sortie
 instance.upgrade.deployed_by = Déployée par
 instance.upgrade.deployment = Déploiement encours
+instance.upgrade.error.abort = Erreur lors de l'abandon de la mise à jour
 instance.upgrade.error.deploy = Erreur lors de la mise à jour
+instance.upgrade.error.deploy_info_fetch = Erreur lors de la récupération des informations de déploiement
+instance.upgrade.error.get_upgrade = Erreur lors de la récupération de la progression de la mise à jour
 instance.upgrade.error.version_fetch = Erreur dans la recherche des versions disponibles
-instance.upgrade.install = Intaller
+instance.upgrade.feature_releases = Versions des fonctionnalités bêta
+instance.upgrade.install = Installer
+instance.upgrade.interrupted = Une erreur inattendue du serveur a provoqué l'interruption de la mise à jour.
 instance.upgrade.no_betas = Il n'y a pas de nouvelles versions bêta vers lesquelles vous pouvez vous mettre à jour.
 instance.upgrade.no_branches = Aucune branche disponible
 instance.upgrade.no_details = (pas de détails) 
+instance.upgrade.no_feature_releases = Il n'existe aucune nouvelle version de fonctionnalité vers laquelle vous pouvez effectuer une mise à jour.
 instance.upgrade.no_horti = Horticulteur requis
 instance.upgrade.no_horti.detail = Pour utiliser cette page, votre instance doit être déployée via un horticulteur.
 instance.upgrade.no_new_releases = Il n'y a pas de nouvelles versions pour une mise à jour
 instance.upgrade.potentiallyIncompatible = Cette version est peut-être incompatible avec votre actuelle version
 instance.upgrade.releases = Versions
+instance.upgrade.retry = Réessayez la mise à jour
 instance.upgrade.stage = Etape
-instance.upgrade.upgrading = Une mise à niveau de {{before}} to {{after}} a été initiée.
+instance.upgrade.state.completing = Fin de la mise à jour
+instance.upgrade.state.complete = Mise à jour terminée
+instance.upgrade.state.finalizing = Finalisation de la mise à niveau
```

```suggestion
instance.upgrade.state.finalizing = Finalisation de la mise à jour
```
"Mise a niveau" est aussi correct mais autant rester consistant avec les autres traductions